src/HybridDb/SqlServerUsingGlobalTempTables.cs

```diff
@@ -12,12 +12,14 @@ namespace HybridDb
     public class SqlServerUsingGlobalTempTables : SqlServer
     {
         SqlConnection schemaBuilderConnection;
+        readonly StoreStats storeStats;
 
         string prefix;
         int numberOfManagedConnections;
 
-        public SqlServerUsingGlobalTempTables(DocumentStore store, string connectionString) : base(store, connectionString)
+        public SqlServerUsingGlobalTempTables(DocumentStore store, string connectionString, StoreStats storeStats) : base(store, connectionString)

```

Comment:
Kan vi ikke tage storeStats direkte fra store i stedet for at sende den ind?

Reply:
Fixed